coordinator/src/admin.rs

```diff
@@ -160,13 +161,18 @@ pub async fn collaborative_revert(
         AppError::BadRequest("Invalid channel id provided".to_string())
     })?;
 
+    let out_point = OutPoint {
```

Comment:
🤔 can't we fetch that from the shadow channel?

Reply:
in theory we could fetch it from a block explorer. But I'm not sure if it is worth the hassle.